Update readme (#1)
nvaccess/addon-store-submission: README.md

- The commit which will be submitted must be a valid NVDA `*.nvda-addon` file if zipped and renamed.

Process:

> 1. Addon author creates a new Pull Request (PR) on the 'NVDA-Addon-submission' repository. This could be done with the web editor, though screen reader users may be more comfortable making it locally with their chosen editor or using a provided template:

Reviewer: Perhaps this should be expanded to include plans for submitting via an issue? If we do that, then I would remove the instructions for how to edit the file, this becomes the 'advanced' workflow for people familiar with the concepts.

Author: I think it would be easier to send issues and remove instructions to edit the file.

     - Check manifest validity
     - Provide links to relevant diffs
     - Check to see if the GitHub user submitting the PR is a maintainer of the Addon version being submitted.

>    - On Appveyor CI, install most recently supported version of NVDA supported by addon, install the addon, restart NVDA and check for errors. If the add-on contains an `installTasks.py` file with an `onInstall()` function which may require input at installation, the `installTasks` file may be skypped to avoid false errors.

Reviewer: I'd prefer not to get into the details of things like this, it's really not core to the proposal, and really just a suggestion of the "type" of things that could be done. Perhaps instead it should just be made more clear that this is a "suggestion".

```suggestion
   - Suggestion: On Appveyor CI, install most recently supported version of NVDA supported by addon, install the addon, restart NVDA and check for errors. 
```

Using a separate repository for this data store separates concerns, gives greater flexibility for managing permissions, makes it easier to verify changes (eg in a PR, or via automation), and simplifies the commit history (no automation commits "updating data store")

> The exact implementation details for this repository are not exposed to users, because on one side the 'NVDA-Addon-submission' repository feeds data in and the NV Access server is on the other side fetching and providing data to NVDA or a web based store.

Reviewer: Good catch.
